e2e/utils/db.ts

```diff
@@ -0,0 +1,41 @@
+import pg from 'pg';
+
+const { Pool } = pg;
+
+class DatabaseConnection {
+  private pool: any;
```

Comment:
[nit] If you npm installed `@types/pg`, you could assign the type `pg.Pool` to this variable to avoid typing errors.